Documentation/security_api.md

```diff
@@ -10,37 +10,32 @@ There are three types of resources in etcd
 ### Permission Resources 
 
 #### Users
-A user is an identity to be authenticated. Each user can have multiple roles. The user has a capability on the resource if one of the roles has that capability.
+A user is an identity to be authenticated. Each user can have multiple roles. The user has a capability (such as reading or writing) on the resource if one of the roles has that capability.
 
-The special static `root` user has a ROOT role. (Caps for visual aid throughout)
+A user named `root` is required before security can be enabled. The ROOT role can be granted to multiple users, but `root` is required for recovery purposes.
 
-#### Role
-Each role has exact one associated Permission List. An permission list exists for each permission on key-value resources. A role with `manage` permission of a key-value resource can grant/revoke capability of that key-value to other roles.
+#### Roles
+Each role has exact one associated Permission List. An permission list exists for each permission on key-value resources. 
+
+The special static ROOT (named `root`) role has a full permissions on all key-value resources, the permission to manage user resources and settings resources. Only the ROOT role has the permission to manage user resources and modify settings resources.
```

Comment:
In order to enable security, root must be created. The error message is helpful.